README.md

```diff
@@ -1,4 +1,12 @@
 # get-NBA
 This is the Capstone-1 project for Springboard Software Engineering Course.
 
-API that I am using: https://www.balldontlie.io/api/v1
+Main API used: https://www.balldontlie.io/api/v1
```

Comment:
There are expected formats for a readme. Everything you have here is awesome, you just missing a little. Check this out
https://github.com/tomaszguzialek/flask-api
This guy has info on how to run, build, install dependencies, run tests. Those are kind of must haves. For you, you want to also include information on how to install, setup, and seed the database locally.

If another developer came here and there weren't directions on this stuff...... 🔥 haha they would be unhappy.

Reply:
Bump on this, gotta have some notes on how to run the project. Your writing this specifically for the next developer so things like `python -m venv venv`, how to activate, how to install deps. Just a few very short line items will save the next developer hours.